src/current/_includes/releases/cloud/2024-04-04.md

```diff
@@ -0,0 +1,26 @@
+## April 4, 2024
+
+<h3 id="2024-04-04-general-updates"> General updates </h3>
+
+- CockroachDB {{ site.data.products.dedicated }} clusters on Azure can now be created in the following [regions]({% link cockroachcloud/regions.md %}):
+
+    Africa          | `southafricanorth`   | Johannesburg
+    Asia Pacific    | `japaneast`          | Tokyo
+                    | `koreacentral`       | Seoul
+                    | `australiaeast`      | New South Wales
```

Comment:
we always supported `australiaeast` and `germanywestcentral`, they can be removed from this list.

Reply:
The full list of regions is given by any org in staging right now.